packages/rum-core/src/domain/contexts/viewContexts.spec.ts

```diff
@@ -1,15 +1,64 @@
-import type { RelativeTime } from '@datadog/browser-core'
+import type { RelativeTime, Duration, TimeStamp } from '@datadog/browser-core'
 import { relativeToClocks, CLEAR_OLD_VALUES_INTERVAL } from '@datadog/browser-core'
 import type { TestSetupBuilder } from '../../../test'
 import { setup } from '../../../test'
 import { LifeCycleEventType } from '../lifeCycle'
-import type { ViewCreatedEvent } from '../view/trackViews'
+import type { ViewCreatedEvent, ViewEvent } from '../view/trackViews'
+import { ViewLoadingType } from '../../rawRumEvent.types'
 import type { ViewContexts } from './viewContexts'
 import { startViewContexts, VIEW_CONTEXT_TIME_OUT_DELAY } from './viewContexts'
 
 describe('viewContexts', () => {
   const FAKE_ID = 'fake'
   const startClocks = relativeToClocks(10 as RelativeTime)
+  const VIEW: ViewEvent = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You probably don't need a full ViewEvent. Something like this should be enough:
```ts
      lifeCycle.notify(LifeCycleEventType.VIEW_UPDATED, {
        startClocks,
        name: 'Fake Name',
      } as ViewEvent)
```